src/manifest.c

```diff
@@ -239,9 +240,20 @@ static struct manifest *manifest_from_file(int version, char *component, bool he
 		}
 		if (strncmp(line, "filecount:", 10) == 0) {
 			filecount = strtoull(c, NULL, 10);
+			if (filecount > 1000000) {
```

Comment:
We should think about the Manifest.full case. Right now there are currently over 427k files listed in Manifest.full for Clear release 21300. We should probably increase the limit here to account for that number increasing in the future.

Reply:
Sure, I actually took that number, doubled it to allow for future growth and then rounded it up to be a nice round number.

We probably shouldn't make it any bigger than 2**32/sizeof(struct file) (currently 176 on a 64/32/64 PIL build), or 12 million because of the hinted at integer overflow issues.

How about 4 million (with a comment) to allow Manifest.full to get an order of magnitude bigger than it currently is?

Reply:
+1 for 4 million with a comment